marketdata/market/dex/dex.go

```diff
@@ -59,9 +59,9 @@ func normalizeTicker(price CoinPrice, provider string) (blockatlas.Ticker, error
 		return blockatlas.Ticker{}, errors.E(err, "normalizeTicker parse value24h error",
 			errors.Params{"PriceChange": price.PriceChangePercent, "Symbol": price.BaseAssetName})
 	}
-	tokenId := price.BaseAssetName
+	tokenId := price.QuoteAssetName
 	if tokenId == BNBAsset {
-		tokenId = price.QuoteAssetName
+		tokenId = price.BaseAssetName
```

Comment:
Is there any issues if one token could be pair to others? 

Like XXX could be listed on markets:
XXX-BNB
XXX-USDC

Reply:
No, this condition solve this problem, but the Dex prices are wrong because the logic is inverted